fs/remote/resolver.go

```diff
@@ -169,25 +170,26 @@ func (r *Resolver) resolveFetcher(ctx context.Context, hosts source.RegistryHost
 }
 
 type fetcherConfig struct {
-	hosts       source.RegistryHosts
-	refspec     reference.Spec
-	desc        ocispec.Descriptor
-	maxRetries  int
-	minWaitMSec time.Duration
-	maxWaitMSec time.Duration
+	hosts      source.RegistryHosts
+	refspec    reference.Spec
+	desc       ocispec.Descriptor
+	maxRetries int
+	minWait    time.Duration
+	maxWait    time.Duration
```

Comment:
I feel it's more intuitive to keep the unit `MSec`. Why do we need to remove this?

Reply:
it's a go-staticcheck violation to put a unit in the identifier of a time.Duration. This member conceptually has no unit, and if you dive into the underlying type then it's actually stored in nanoseconds, not milliseconds. https://staticcheck.io/docs/checks#ST1011